package.json

```diff
@@ -14,11 +14,20 @@
     "owners-file": "https://github.com/bkeepers/owners-node.git"
   },
   "devDependencies": {
+    "expect": "^1.20.2",
+    "mocha": "^3.2.0",
     "xo": "^0.16.0"
   },
   "xo": {
     "esnext": true,
-    "space": true
+    "space": true,
+    "envs": [
+      "node",
+      "mocha"
+    ],
+    "rules": {
+      "camelcase": 1
+    }
```

Comment:
Yeah, that would make a lot of sense :+1:

Or perhaps a probot plugin generator that drops a standard template like the [Atom package-generator package](https://github.com/atom/package-generator)? Maybe just make it a probot command-line command?